src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -262,7 +262,7 @@ public boolean equals(Object other) {
      */
     public Predicate<Person> getPersonLinkedToCompanyPredicate(Company targetCompany) {
         return person -> person.isMatchPresent()
-                && person.getMatch().startsWith(targetCompany.getName().toString() + "::");
+                && person.getMatch().orElse("").startsWith(targetCompany.getName().toString() + "::");
```

Comment:
I am not sure why this is necessary? If `isMatchPresent()` is true, we won't need the `orElse` since we can always `get` right?

Reply:
We can also use `get`, I just like using `orElse` cause getMatch.get sounds worse XD. But okay I will utilise your suggestion.